Add ssh host CA roles

jabl opened the pull request:
ansible-role-ssh-hostca-srv role to install, it configures and runs
a ssh host CA signing service.

ansible-role-sshd-host-keys-client to all other nodes, which contacts
the host CA service and gets a signed certificate.

local.yml left alone for now until sites have updated install node.

martbhell commented:
I haven't looked at the contents of the PR - but travis fails with

<pre> - extracting ansible-role-ssh-hostca-srv to /ansible-role-ssh-hostca-srv
[0;31mERROR! Unexpected Exception: [Errno 21] Is a directory: '/ansible-role-ssh-hostca-srv/tasks'[0m
to see the full traceback, use -vvv
requirements install failed</pre>

A1ve5 commented:
Looks good. We can test it in Io.
@jabl, one question why not to have the python server script included in the role? (https://github.com/jabl/ansible-role-ssh-hostca-srv/blob/master/tasks/main.yml#L40)

jabl commented:
Ok, I think I fixed that weird "Errno[21] is a directory" error. Turned out the repo accidentally contained an emacs backup file (tasks/main.yml~). Removed that and pushed a new tag, lets see if this works better.

As for why the script is downloaded from a separate repo instead of included under files/. Well, no particular reason, really. I just started developing the script in a separate repo before ansiblifying it. Might make sense to tag a version of that repo and download the tagged version, so one doesn't accidentally download some in-progress development work..

jabl commented:
Latest version now using tagged version of ssh_hostca_srv.py script.

A1ve5 commented on install.yml:
```diff
@@ -62,4 +64,4 @@
   - { role: ansible-role-collectd, tags: [ 'collectd', 'monitoring' ] }
   - { role: ansible-role-flowdock, tags: [ 'flowdock', 'always' ] }
-
+  - { role: ansible-role-sshd-host-keys-client, tags: [ 'ssh', 'ssh-hostca' ] }
```
@jabl you want it to run twice?

jabl replied:
No, the second time is when running against admin after running against install. Can't put the host-keys-client role directly into admin.yml because then it would fail before the server part is setup on install.

A1ve5 replied:
Ah. My bad, I should have expanded the view... now it makes sense.

A1ve5 commented on requirements.yml:
```diff
@@ -203,3 +203,11 @@
+
+- src: https://github.com/jabl/ansible-role-ssh-hostca-srv
+  path: roles
+  version: v1.0.2
```
Yes, I like it more with a tagged version. Thanks!

rkdarst commented:
Can this old PR from 2016 be closed now?

I'm looking at old PRs from my organizations, and this is a very old PR, and I guess it can be closed now.  But I can't do it myself.

VilleS1 commented:
Yeah lets close it.
